src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/pdl/PersonService.kt

```diff
@@ -79,8 +79,15 @@ class PersonService(
 
     fun hentAndreForeldre(personIdenter: List<String>): Map<String, PdlAnnenForelder> = pdlClient.hentAndreForeldre(personIdenter)
 
-    @Cacheable("personidenter")
-    fun hentPersonIdenter(ident: String): PdlIdenter = pdlClient.hentPersonidenter(ident = ident)
+    fun hentFolkeregisterIdenter(ident: String): PdlIdenter = hentIdenterCached(ident).folkeregisteridenter()
+
+    fun hentFolkeregisterOgNpidIdenter(ident: String): PdlIdenter =
+        hentIdenterCached(ident).medIdentgrupper(PdlIdentGruppe.FOLKEREGISTERIDENT.name, PdlIdentGruppe.NPID.name)
+
+    private fun hentIdenterCached(ident: String): PdlIdenter =
+        cacheManager.getValue("personidenter", ident) {
+            pdlClient.hentPersonidenter(ident)
+        }
```

Comment:
Ble litt krøll med caching etter refaktorering. 

Forsøkte først å flytte `Cacheable`-annotasjonen til `PDLClient.hentPersonidenter`, men siden vi eksponerer en mockk-instans i integrasjonstestene, førte det til en del feil (kort fortalt; spring lager en proxy rundt `PDLClient` når den blir eksponert som en spring-bønne når den har en funksjon med `Cacheable`-annotasjon, og da klarer ikke mockK å kjenne igjen at det er en mock, ved `resetMock`-kall i integrasjonstester).

Håndterer derfor cachen manuelt her, men med en endring i libs https://github.com/navikt/tilleggsstonader-libs/pull/95 for at evt exception skal propageres riktig